language/block_spec.rb

```diff
@@ -1001,55 +1001,53 @@ def a; 1; end
 
 # tested more thoroughly in language/delegation_spec.rb
 describe "Anonymous block forwarding" do
-  ruby_version_is "3.1" do
-    it "forwards blocks to other method that formally declares anonymous block" do
-      eval <<-EOF
-          def b(&); c(&) end
-          def c(&); yield :non_null end
-      EOF
+  it "forwards blocks to other method that formally declares anonymous block" do
+    eval <<-EOF
+        def b(&); c(&) end
+        def c(&); yield :non_null end
+    EOF
```

Comment:
Does it make sense to remove `eval` calls in touched the specs for 3.1 syntax changes now?

Reply:
yes